src/components/Filter/Filter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function Filter( {filters, active, setActive }) {
+    return (
+    <ButtonWrapper>
+    {filters.map((filter) => (
+    <FilterButton key={filter} active={active === filter} onClick={() => activeFilter(filter)}>{filter.includes('all') ? filter : 'Hide ' + filter}</FilterButton>))}
+    </ButtonWrapper>
+    )
+
+    function activeFilter(filter){
+        setActive(filter)
+    }
+}
+
+
+const ButtonWrapper = styled.section`
+    grid-column: 1/3;
+    display: flex;
+    margin-bottom: 20px;
+`
+
+const FilterButton = styled.button`
+    width: 85px;
+    height: 35px;
+    border: 2px solid #4BDB80;
+    color: ${({active}) => active ? 'white' : '#4BDB80'};
```

Comment:
You could stay with one convention
```suggestion
    color: ${({active}) => active ? '#FFF' : '#4BDB80'};
```